src/main/java/Duke.java

```diff
@@ -1,10 +1,68 @@
+import static ui.Ui.echo;
+import static ui.Ui.welcome;
+
+import parser.Parser;
+import storage.Storage;
+import tasklist.TaskList;
+
 public class Duke {
+    private Storage stores;
```

Comment:
You can consider using a better/more intuitive variable name for the storage field in Duke